src/Hazelcast.Net/Serialization/Compact/CompactWriterExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Will it be filled in other PRs?

Reply:
yes I'll do one general cleanup of all headers eventually - don't want to fix each PR one by one

Reply:
Actually I meant the file not the header. The position of comment is misleading, sorry. The file is empty. I wondered the reason.

Reply:
oh? you're right. i created this file along with the `CompactReaderExtensions.cs` file but it never got anything into it.
removing it.